org.adempiere.base/src/org/compiere/util/CacheMgt.java

```diff
@@ -373,7 +373,7 @@ public int getElementCount()
 		CacheInterface[] instances = getInstancesAsArray();
 		for (CacheInterface stored : instances)
 		{
-			if (stored != null && stored.size() > 0)
+			if (stored != null && stored instanceof CCache)

```

Comment:
we already has instanceof check below and there's nothing wrong with the size() > 0 check. sizeNoExpire() will never return > 0 if size() is 0.